producer.go

```diff
@@ -289,6 +293,11 @@ func (opts *ProduceOpts) produce(p *Producer) error {
 func (p *Producer) validateMsg(msg any) ([]byte, error) {
 	sd, err := p.getSchemaDetails()
 	if err != nil {
+		failMsg := SchemaFailMsg{
+			Msg: "Schema validation has failed at station " + p.stationName + " with producer " + p.Name + ": " + err.Error(),
+		}
+		msgToSlack, _ := json.Marshal(failMsg)
+		p.conn.brokerConn.Publish("$memphis_schema_validation_fail_updates", msgToSlack)
```

Comment:
This piece repeats itself 3 times, we should make a function out of it

in the future we may extend this function to serve more integrations when this failure happens

Reply:
`$memphis_schema_validation_fail_updates` should be a constant